DailyQuickstart/ViewController.swift

```diff
@@ -0,0 +1,157 @@
+import UIKit
+import Daily
+
+final class ViewController: UIViewController {
+    // The client used to manage the video call.
+    private let callClient: CallClient = .init()
+
+    // The local participant video view.
+    private let localVideoView: VideoView = .init()
+
+    // A dictionary of remote participant video views.
+    private var videoViews: [ParticipantId: VideoView] = [:]
+
+    // The URL for the room to join.
+    private let roomURLString: String = "[YOUR_DAILY_ROOM_URL]"
+
+    // MARK: - Buttons
+
+    @IBOutlet private weak var cameraInputButton: UIButton!
+    @IBOutlet private weak var microphoneInputButton: UIButton!
+    @IBOutlet private weak var leaveRoomButton: UIButton!
+
+    // MARK: -
```

Comment:
The `// MARK: -` syntax will create horizontal rules in the jump bar available via `ctrl-6` in Xcode, which helps group related properties visually.